client/extensions/woocommerce/app/dashboard/manage-no-orders.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import BasicWidget from '../../components/basic-widget';
+import { getLink } from '../../lib/nav-utils';
+import ReadingWidget from '../../components/reading-widget';
+import ShareWidget from '../../components/share-widget';
+import WidgetGroup from '../../components/widget-group';
+
+class ManageNoOrders extends Component {
+	static propTypes = {
+		site: PropTypes.shape( {
+			slug: PropTypes.string.isRequired,
+		} ),
+	};
+
+	renderShareWidget = () => {
+		const { site, translate } = this.props;
+		return (
+			<ShareWidget
+				text={ translate( 'Congratulations! Now that your setup is complete you can spread the word' +
+					' and get those first orders rolling in. Share your store with friends,' +
+					' family, and followers now.' ) }
+				title={ translate( 'Your store is ready, the world awaits!' ) }
+
+				urlToShare={ getLink( 'https://:site', site ) }
+			/>
+		);
+	}
+
+	renderReadingWidget = () => {
+		const { site, translate } = this.props;
+		return (
+			<ReadingWidget
+				className="dashboard__reading-widget"
+				site={ site }
```

Comment:
Thank you! Good catch! Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/5512a8530a3394de9a0cedd0beb566762edddebc